scripts/functions

```diff
@@ -1305,6 +1305,25 @@ CT_DoBuildTargetTuple() {
     CT_ARCH_TARGET_LDFLAGS="${CT_ARCH_TARGET_LDFLAGS} ${CT_ARCH_ENDIAN_LDFLAG}"
 }
 
+# This function determines the target tuple for a given set of compiler
+# flags, using either GCC's multiarch feature (if supported; if not,
+# GCC prints nothing and exits with status 0), falling back to calling
+# the architecture-specific functions.
+CT_DoMultilibTarget() {
+    local target="$1"; shift
+    local -a multi_flags=( "$@" )
+    local gcc_multiarch
+
+    gcc_multiarch=$( "${CT_TARGET}-gcc" -print-multiarch "${multi_flags[@]}" )
```

Comment:
For good measure, I'd have redirected `stderr` to `/dev/null`,
just in case different versions of gcc behaves differently.

This can however be changed later, if that happens to be the case.